internal/translation/teams/teams.go

```diff
@@ -0,0 +1,130 @@
+package teams
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+)
+
+type TeamsService interface {
+	ListProjectTeams(ctx context.Context, projectID string) ([]AssignedTeam, error)
+	GetTeamByName(ctx context.Context, orgID, teamName string) (*AssignedTeam, error)
+	GetTeamByID(ctx context.Context, orgID, teamID string) (*AssignedTeam, error)
+	Assign(ctx context.Context, at *[]AssignedTeam, projectID string) error
+	Unassign(ctx context.Context, projectID, teamID string) error
+	Create(ctx context.Context, at *Team, orgID string) (*Team, error)
+	RenameTeam(ctx context.Context, at *AssignedTeam, orgID, newName string) (*AssignedTeam, error)
+	UpdateRoles(ctx context.Context, at *AssignedTeam, projectID string, newRoles []akov2.TeamRole) error
+
+	GetTeamUsers(ctx context.Context, orgID, teamID string) ([]TeamUser, error)
+	AddUsers(ctx context.Context, usersToAdd *[]admin.AddUserToTeam, orgID, teamID string) error
```

Comment:
The `admin.` types should not leak out of the translation layer.

```suggestion
	AddUsers(ctx context.Context, users []TeamUser, orgID, teamID string) error
```

Reply:
Noted! Fixing now